### PR TITLE
shows messages on chat page

### LINKED ISSUE #47: Show the messages on the chat page.
Currently we have implemented the basic functionality of our app but the messages are still shown in console. Please show them on chat page. Make sure the messages pop up.

### PR DESCRIPTION
**Pull request by jainkeval**
fixes #47 
https://morning-reef-65656.herokuapp.com/

### CODE REVIEW

File: socketserver.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = (io, app) => {
   io.on('connection', (socket) => {
     // Make a user object and add it to the onlineUsers list and rooms too(maybe we can add to room once we have the partner.)
     socket.emit('ack', { id: socket.id, msg: "User connected" });
-
+    // console.log('fuck ir ' + socket.id);
```
Comment (sumedh123) [inline socketserver.js]: remove this.

File: views/chat.hbs [hunk 2]
```diff
@@ -58,8 +46,8 @@
     </div>
   </div>
 </body>
-<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script> -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (sumedh123) [inline views/chat.hbs]: why?

File: socketserver.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ module.exports = (io, app) => {
     });
 
     socket.on('sendMessage', (data) => {
-      io.sockets.in(data.room).emit('newMessage', { "message": data.message });
+      io.sockets.in(data.room).emit('newMessage', { "message": data.message , "sender_id": socket.id});
```
Comment (sumedh123) [inline socketserver.js]: camel case

Comment (jainkeval) [reply socketserver.js]: Sure!

File: views/chat.hbs (hunk 2 above)
Comment (jainkeval) [reply views/chat.hbs]: I used jquery to append messages

Comment (jainkeval) [reply views/chat.hbs]: ohkkk

**Comment by kvnamipara** [comment]
@sumedh123 close the PR if motive is full filled

**Comment by sumedh123** [comment]
@kvnamipara This is already merged.

**Pull request merged**